books/tools/soft-paper-examples.lisp

```diff
@@ -1,414 +1,3 @@
-; SOFT Examples from the ACL2-2015 Workshop paper
-;
-; Copyright (C) 2015 Kestrel Institute (http://www.kestrel.edu)
-;
-; License (an MIT license):
-;
-;   Permission is hereby granted, free of charge, to any person obtaining a
-;   copy of this software and associated documentation files (the "Software"),
-;   to deal in the Software without restriction, including without limitation
-;   the rights to use, copy, modify, merge, publish, distribute, sublicense,
-;   and/or sell copies of the Software, and to permit persons to whom the
-;   Software is furnished to do so, subject to the following conditions:
-;
-;   The above copyright notice and this permission notice shall be included in
-;   all copies or substantial portions of the Software.
-;
-;   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-;   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-;   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-;   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-;   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-;   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
-;   DEALINGS IN THE SOFTWARE.
-;
-; Original author: Alessandro Coglio (coglio@kestrel.edu)
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
-; This file contains the SOFT ('Second-Order Functions and Theorems') examples
-; in the ACL2-2015 Workshop paper "Second-Order Functions and Theorems in ACL2".
-
-; Comments indicate the sections and subsections of the paper.
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
 (in-package "ACL2")
 
-(include-book "soft")
```

Comment:
Hi, David --

I don't see anything in the acl2+books manual about relocation stubs.
Could you add something about that, which could point to your wiki
page?  Probably I'm not the only ACL2 user who tends not to look for
wiki pages, instead expecting to find pretty much everything in the
manual other than what's easily accessible from the ACL2 home page.

I'm willing to help, but I know very little about such infrastructure,
which is why I'm asking you....

Thanks --
-- Matt
> Date: Sun, 17 Jan 2016 07:31:43 -0800
> From: "David L. Rager" <notifications@github.com>
> Reply-To: acl2/acl2 <reply@reply.github.com>
> 
> 
> [1:text/plain Hide]
> 
> >  (in-package "ACL2")
> >  
> > -(include-book "soft")
> 
> It's unclear whether soft should be in kestrel or a more general place.  Anyway, I'm happy to have it moved to kestrel for now.  Do you mind creating another pull request that tags these as "relocation stubs"?  We didn't have this in a well-documented place, so I added it to the wiki.
> https://github.com/acl2/acl2/wiki/How-to-move-a-book
> 
> Also, your CL highlights our thus-far-unstructured desire to come up with a structure for testing outside a normal build -- I think you're pretty close to what we want already. :)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/acl2/acl2/pull/576/files#r49947737
> 
> [2:text/html Show]
>